ui/src/shared/components/ConfirmButtons.js

```diff
@@ -15,20 +21,23 @@ const ConfirmButtons = ({onConfirm, item, onCancel, buttonSize}) => (
       className={classnames('btn btn-success btn-square', {
         [buttonSize]: buttonSize,
       })}
+      disabled={isDisabled}
```

Comment:
If `isSavable` is passed through as `true`, the button would be `disabled` – this seems opposite to what's desired, no? As far as I can tell, the logic needs to be reversed at the source, at Line 186 of CellEditorOverlay.js (hence my comments there) in order to pass through as intended and to not affect other `<ConfirmButtons>` components.